project/Dist.scala

```diff
@@ -24,8 +24,10 @@ object Dist {
   val distDocJars = TaskKey[Seq[File]]("dist-doc-jars")
   val distSources = TaskKey[DistSources]("dist-sources")
   val dist = TaskKey[File]("dist", "Create a zipped distribution of everything.")
+  val includeInDist = SettingKey[Boolean]("include-in-dist", "Include the artifact of this project in the standalone dist zip-file")
 
   lazy val settings: Seq[Setting[_]] = Seq(
+    includeInDist := true,
```

Comment:
Yeah, was part of `rootSettings` which I incorrectly assumed was inherited by the modules